CHANGELOG.md

```diff
@@ -0,0 +1,176 @@
+# Quarkus extension for MicroProfile Language Server Changelog
+
+## 0.9.0 (September 21, 2020)
+
+Since `0.9.0`, quarkus-ls becomes an extension of [lsp4mp](https://github.com/eclipse/lsp4mp), see [CHANGELOG of vscode-lsp4mp](https://github.com/eclipse/lsp4mp/CHANGELOG.md#010-september-14-2020)

```

Comment:
becomes an extension of [lsp4mp](https://github.com/eclipse/lsp4mp), the Language Server for Eclipse MicroProfile, see the upstream [lsp4mp CHANGELOG] for additional informations about enhancements and bug fixes.

Reply:
should link to the sept 21 anchor of lsp4mp's changelog

Reply:
fixed